db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
```

Comment:
- [ ] Kindly store 5 different greetings in your seeds.rb that will be displayed when a user reloads your frontend application, this will also clear the error seen on your frontend application👍🏿